index.bs

```diff
@@ -746,29 +747,28 @@ authorizing an authenticator.
 </div>
 
 
-### Use an existing credential to make an assertion - PublicKeyCredential's `[[DiscoverFromExternalSource]](options)` method ### {#getAssertion}
+### Use an existing credential to make an assertion ### {#getAssertion}
```

Comment:
I'm not sure about pulling the "PublicKeyCredential's `[[DiscoverFromExternalSource]](options)` method" text out of the section title. if we do that we need to update all the other section titles to adopt the same style (IMV).  I prefer having it in the title beause it yields hits in the ToC.

Reply:
I do generally prefer short section titles, but I took the [[DiscoverFromExternalSource]] out of this section name for a different reason: this section discusses more than just that method. I'm inclined to make a sub-section if you really want the method name in the ToC?

Reply:
yeah, we have most all interface/dict/object/whathaveyou names in the toc (which I, at least, find useful) so i'd be inclined to have it in there one way or another. I spose this style of change would also apply to "[Create a new credential - PublicKeyCredential’s \[\[Create\]\](options) method](https://w3c.github.io/webauthn/#createCredential)"  since it also discusses "more than just that method" ([IIUC](https://en.wiktionary.org/wiki/IIUC)) ?

Reply:
I've added the sub-section, but I can put it back in the "Use an existing credential" if you prefer. Note that "Create a new credential" starts with the definition of [[Create]], while ["Use an existing credential"](https://s3.amazonaws.com/pr-preview/jyasskin/webauthn/clarify-get-flow.html#getAssertion) starts with a description of `credentials.get()`.